Competition2018/src/test/java/competition/subsystems/climberdeploy/Command/Tests/DetractClimberArmCommandTest.java

```diff
@@ -0,0 +1,39 @@
+package competition.subsystems.climberdeploy.Command.Tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import competition.subsystems.climberdeploy.ClimberdeploySubsystem;
+import competition.subsystems.climberdeploy.commands.DetractClimberArmCommand;
+import xbot.common.controls.actuators.mock_adapters.MockCANTalon;
+import xbot.common.injection.BaseWPITest;
+
+public class DetractClimberArmCommandTest extends BaseWPITest{
```

Comment:
I missed this before, but it should be `RetractClimberArmCommand` , not `Detract`

Reply:
Refactored and renamed methods .